### DISCUSSION TITLE
Add categories

### DISCUSSION THREAD

**Discussion by IntnsRed** [PaulWoitaschek/Voice#1356] [Ideas]
First, great program! Kudos.

I have a couple of dozen audio books on my Android-powered MOTSCO (MObile Tracking and Surveillance COmputer; aka "cell phone"). I love how Voice remembers where I left off when doing several books at once.

But what's missing is the ability to put books into categories. Users should be able to create/delete categories and put books into various folders/categories.

This might be done by doing multiple "Audiobook folders" with each folder being a separate category, or perhaps using sub-directories on the Android/Linux file system as the categories.

This moves away a bit from Voice's description of being a "Minimalistic audiobook player" but the add'l functionality would be nice _IMO._ :-)

**Comment by mesonium** [comment]
Duplicate of https://github.com/PaulWoitaschek/Voice/discussions/1214 or https://github.com/PaulWoitaschek/Voice/discussions/1314
